packages/beacon-node/src/chain/blocks/importBlock.ts

```diff
@@ -314,9 +322,53 @@ export async function importBlock(
   // Notifying EL of head and finalized updates as below is usually done within the 1st 4s of the slot.
   // If there is an advanced payload generation in the next slot, we'll notify EL again 4s before next
   // slot via PrepareNextSlotScheduler. There is no harm updating the ELs with same data, it will just ignore it.
+
+  // Suppress fcu call if shouldOverrideFcu is true. This only happens if we have proposer boost reorg enabled
+  // and the block is weak and can potentially be reorged out.
+  let shouldOverrideFcu = false;
+  let notOverrideFcuReason = NotReorgedReason.Unknown;
+
+  if (opts.isGossipBlock && isExecutionStateType(postState)) {
+    try {
+      const proposerIndex = postState.epochCtx.getBeaconProposer(blockSlot + 1);
+      const feeRecipient = this.beaconProposerCache.get(proposerIndex);
+
+      if (feeRecipient) {
+        // We would set this to true if
+        //  1) This is a gossip block
+        //  2) We are proposer of next slot
+        //  3) Proposer boost reorg related flag is turned on (this is checked inside the function)
+        //  4) Block meets the criteria of being re-orged out (this is also checked inside the function)
+        const result = this.forkChoice.shouldOverrideForkChoiceUpdate(this.clock.currentSlot, blockSummary.blockRoot);
+        shouldOverrideFcu = result.shouldOverrideFcu;
+        if (!result.shouldOverrideFcu) {
+          notOverrideFcuReason = result.reason;
+        }
+      }
+    } catch (e) {
+      if (isStartSlotOfEpoch(blockSlot + 1)) {
+        notOverrideFcuReason = NotReorgedReason.NotShufflingStable;
+      } else {
+        this.logger.warn("Unable to get beacon proposer. Do not override fcu.", {slot: blockSlot + 1}, e as Error);
```

Comment:
`slot` here might be misleading as it's the proposal slot, not block slot or clock slot

maybe add const `proposalSlot` as we do `blockSlot + 1` in multiple places
```suggestion
        this.logger.warn("Unable to get beacon proposer. Do not override fcu.", {proposalSlot}, e as Error);
```